purchase_request_department/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+{
+    "name": "Purchase Request Department",
+    "author": "Eficent, "
```

Comment:
IMO Add `website` key also as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)